include/psopt.h

```diff
@@ -30,8 +30,8 @@ e-mail:    v.m.becerra@ieee.org
 using std::numeric_limits;
 
 namespace PSOPT {
-    constexpr double inf        =   std::numeric_limits<double>::infinity();
-    constexpr double pi         =   4.0*atan(1.0);
+    constexpr double inf    = std::numeric_limits<double>::infinity();
+    const double pi         = 4.0*atan(1.0);
```

Comment:
On newer compilers constexpr is enforced and atan can't be calculated at compiletime with this c++ version.